website/docs/justmycode.md

```diff
@@ -0,0 +1,33 @@
+Enable or disable support for [JustMyCode](https://docs.microsoft.com/en-us/visualstudio/debugger/just-my-code?view=vs-2019) debugging in your native C++ code.
```

Comment:
And `justmycode` seems not a best api name because `justmycode` maps to `Debug > Options > Debugging > General > Enable Just My Code`. But this api is used to control if current project will link `__CheckForDebuggerJustMyCode` from C Runtime Library or not.
`supportjustmycode` looks better for me.